cypress/e2e/default/learn/common-components/search-bar.ts

```diff
@@ -106,6 +106,6 @@ describe('Search bar', () => {
 
     cy.get('.ais-SearchBox-form').submit();
 
-    cy.url('/');
+    cy.url().should('eq', 'http://localhost:8000/learn/');
```

Comment:
```suggestion
    cy.location().should(loc => {
      expect(loc.pathname).to.eq('/learn');
    });
```

Reply:
Cool, that worked when testing on my machine 👍

Side question: As a general approach, is it sufficient to run tests like `npm run cypress -- run --spec=cypress/e2e/default/learn/common-components/search-bar.ts`, or should the entire test suite be ran before submitting a PR? (`npm run cypress:dev:run`)

Reply:
If you are changing test files, it is sufficient to run just the specs you are changing.

If you are changing client/server code, it is best to run the whole suite (at some point).